### ISSUE TITLE
[BUG] Move cursor to start of the list item in creating a new list

### ISSUE THREAD

**Issue by chaitanya-scio** [HarryChen0506/react-markdown-editor-lite#208]
## OS and browser
macOS 11.3, Chrome 90.

## Description
When hitting any of the list buttons to create a new list, the cursor is not moved to the start of the list point. 
![image](https://user-images.githubusercontent.com/54037984/120036431-faf04180-bfb4-11eb-8071-456ab71e7e02.png)
This might be a better UX that people expect?

## Reproduction URL
At the demo itself: https://harrychen0506.github.io/react-markdown-editor-lite/

**Comment by sylingd** [comment]
Not implemented yet, already planned

**Comment by sylingd** [comment]
I haver release react-markdown-editor-lite@1.2.5-5. please upgrade and test

**Comment by chaitanya-scio** [comment]
Thank you!!
Trying to test with the upgrade but failing to compile, if you have any clues:
```
ERROR in ./node_modules/react-markdown-editor-lite/es/components/Icon/fonts/iconfont.css
Module build failed (from ./node_modules/mini-css-extract-plugin/dist/loader.js):
ModuleParseError: Module parse failed: Unexpected token (1:0)
```

**Comment by sylingd** [comment]
> Thank you!!
> Trying to test with the upgrade but failing to compile, if you have any clues:
> 
> ```
> ERROR in ./node_modules/react-markdown-editor-lite/es/components/Icon/fonts/iconfont.css
> Module build failed (from ./node_modules/mini-css-extract-plugin/dist/loader.js):
> ModuleParseError: Module parse failed: Unexpected token (1:0)
> ```

Try 1.2.5-6

**Comment by chaitanya-scio** [comment]
Yeah that works compilation-wise and the cursor seems to be fixed! As a separate issue, the table icons might be a little messed up on this v:
![image](https://user-images.githubusercontent.com/54037984/123495955-9ee00380-d5da-11eb-8c09-499843b35e18.png)

**Comment by sylingd** [comment]
> Yeah that works compilation-wise and the cursor seems to be fixed! As a separate issue, the table icons might be a little messed up on this v:
> ![image](https://user-images.githubusercontent.com/54037984/123495955-9ee00380-d5da-11eb-8c09-499843b35e18.png)

fixed at 1.2.5-8

**Comment by chaitanya-scio** [comment]
Thank you so so so much for all your help for the original issue and everything afterwards!!!! Looks great!
